Source/Delphi.Mocks.Helpers.pas

```diff
@@ -33,9 +33,20 @@
 interface
 
 uses
-  Rtti;
+  System.Generics.Collections,
+  System.Rtti;
 
 type
+  //Allow custom comparisons
+  TCustomValueComparer = reference to function(const a, b: TValue): Integer;
+  TCustomValueComparerStore = record
+  private
+    class var CustomComparers: TDictionary<Pointer, TCustomValueComparer>;
```

Comment:
could this not be 
````class var CustomComparers: TDictionary<PTypeInfo, TCustomValueComparer>;````
ie more strongly typed.

Reply:
Yeah, good point, will update that.